docset/winserver2022-ps/failoverclusters/Set-ClusterAffinityRule.md

```diff
@@ -0,0 +1,260 @@
+---
+description: Set-ClusterAffinityRule
+external help file: ClusterAffinityRule.cdxml-help.xml
+Module Name: FailoverClusters
+ms.date: 07/27/2023
+online version: https://learn.microsoft.com/powershell/module/failoverclusters/set-clusteraffinityrule?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: Set-ClusterAffinityRule
+---
+
+# Set-ClusterAffinityRule
+
+## SYNOPSIS
+Enable or disable an affinity rule, and update the rule type.
+
+## SYNTAX
+
+### Query (cdxml) (Default)
+
+```
+Set-ClusterAffinityRule [[-Name] <String[]>] [-RuleType <RuleType>] [-Enabled <UInt32>]
+ [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob] [-PassThru] [<CommonParameters>]
+```
+
+### InputObject (cdxml)
+
+```
+Set-ClusterAffinityRule -InputObject <CimInstance[]> [-RuleType <RuleType>] [-Enabled <UInt32>]
+ [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>] [-AsJob] [-PassThru] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+The `Set-ClusterAffinityRule` command can enable or disable an existing affinity rule. It can also
+change the **RuleType**.
+
+## EXAMPLES
+
+### Example 1 - Enable an affinity rule
+
+```powershell
+Set-ClusterAffinityRule -Name AffinityRule1 -Enabled
+```
+
+This example enables the cluster affinity rule named `AffinityRule1`.
+
+### Example 2 - Disable an affinity rule
+
+```powershell
+Set-ClusterAffinityRule -Name AffinityRule1 -Enabled:$false
```

Comment:
Same point as above
```suggestion
Set-ClusterAffinityRule -Name AffinityRule1 -Enabled $false
```

Reply:
This did work interestingly. I've accepted the suggestion for consistency though!